---
layout: archive
title: "Projects"
permalink: /projects/
author_profile: true
redirect_from:
  - /resume
---

ðŸŒŸThe genomic diversity and adaptation of eukaryotic picophytoplankton Mamiellales
======
I have a longstanding interest in marine phytoplankton. During my doctoral program, I isolated more than 300 Mamiellales strains from the South China Sea using strategies like selective media, flow cytometry, and plaque assays. I have purified and characterized several new clades or species from those cultures, including _Ostreococcus_ Clade E and F, _Bathycoccus_ Clade BIII, _Mantoniella tinhauana_. Leveraging long-read whole genome sequencing and global metagenomic data, I am looking into their biogeography, diversification, adaptation, and evolution. Part of the findings have been published in peer-reviewed journal articles (Xu et al., ISME J, 2024; Rey Redondo et al., Front. Microbiol., 2024).

ðŸŒŸInsights into the phytoplankton-virus interactions through Mamiellales-Prasinovirus model system
======
Having established algal cultures, I then sampled plenty of samples from various marine environments to isolate prasinoviruses from the Mamiellales hosts. We are examining the structural characteristics, decoding the genetic makeup, and elucidating the infection dynamics of these viruses. These Mamiellales-Prasinovirus model systems that we have developed offer deep insights into complex interactions between giant viruses and eukaryotic phytoplankton. One collaborative study with the GENOPHY group at the Observatoire OcÃ©anologique de Banyuls has been published (Thomy et al., ISME common., 2024).

ðŸŒŸTracking virus-host dynamics in microbial communities from natural seawaters.
======
Linking Virus-host pairs in natural communities is still one of the main open questions in environmental microbiology. In this project, I am investigating the seasonal dynamics of bacteriophages, eukaryotic viruses, and RNA viruses through our monthly metagenomic and metatranscriptomic data from coastal water samples. Utilizing state-of-the-art metagenomic Hi-C technology in combination with multiple virus-host prediction approaches, such as sequence homology and CRISPR loci search, I aim to uncover substantial virus-host interactions directly from these microbial ecosystems and shed light on the ecological implications of virus-host dynamics.

ðŸŒŸGenome-resolved insights into the diversity and functions of microbial community in subtropical waters
======
Reconstruction of microbial genomes directly from complex communities poses another significant challenge, with the recovery of eukaryotic genomes being particularly difficult. To solve this question, we have introduced long-read sequencing, Hi-C sequencing, and the latest binning algorithms into time-series metagenomics to acquire high-quality metagenome-assembled genomes (MAGs). These MAGs, belonging to prokaryotes, eukaryotes, and viruses, are helping us to create a high-resolution, multi-dimensional picture of marine microbial communities (Xu et al., Sci. Total Environ., 2024). This research provides crucial insights into ocean food webs, global nutrient cycles, and the response of microbial communities to environmental changes and human influences.
